src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs

```diff
@@ -3200,11 +3391,11 @@ public async Task TestAbstractBase()
 
 abstract class Base
 {
-    public abstract override bool Equals(object? obj);
+    public abstract override bool Equals(object{|CS8632:?|} obj);
```

Comment:
💡 Either add `#nullable enable` to the top of the file, or remove the `?` annotation.

Reply:
done.